tools/tinify_images.py

```diff
@@ -2,16 +2,29 @@
 import pathlib
 import tinify
 import time
-
-
-def tinify_images(path, extensions):
-
-    for file_path in [file for file in pathlib.Path(path).rglob('*')]:
-        if file_path.suffix in extensions:
-            print(f'Compressing image {file_path}')
-            source = tinify.from_file(file_path)
-            source.to_file(file_path)
+import os
+
+
+def tinify_images(target, extensions):
+
+    if os.path.isdir(target):
+        for file_path in [file for file in pathlib.Path(path).rglob('*')]:
+            if file_path.suffix in extensions:
+                print(f'Compressing image {file_path}')
+                source = tinify.from_file(file_path)
+                source.to_file(file_path)
+                time.sleep(0.1)
+    elif os.path.isfile(target):
```

Comment:
We should test this or previous solution depending on how it works with paths that have to expanded, such as `~/dev/orange3`.